ludwig/schema/combiners/base.py

```diff
@@ -1,8 +1,11 @@
+from marshmallow_dataclass import dataclass
+
 from ludwig.api_annotations import DeveloperAPI
 from ludwig.schema import utils as schema_utils
 
 
 @DeveloperAPI
+@dataclass(repr=False, order=True)
```

Comment:
Can we import and use ludwig_dataclass here instead?

```
from ludwig.schema.utils import ludwig_dataclass

@ludwig_dataclass
```

Reply:
Done. What about `BaseFeatureConfig` and the other classes in that file? Does it matter that `order=True` isn't specified on those?

Reply:
Ya we should use `@ludwig_dataclass` on those as well. Currently the it's not a huge deal since a lot of the parameters on those classes are filtered out or not shown for one reason or another. However, we will need it eventually and having `order=True` doesn't hurt so I think adding it in is a great idea.